JASP-Engine/JASP/R/commonbayesianlinearmodels.R

```diff
@@ -86,6 +86,16 @@
 		error.message <- "Bayes factor is undefined -- the dependent variable contains infinity"
 
 	if (!exists ("error.message") && perform == "run") {
+		covariate.names <- NULL
+		for (covariate in options$covariates) {
+			if (any (!is.finite (dataset [[.v (covariate)]])))
+				covariate.names <- c (covariate.names, covariate)
+		}
+		if ( !is.null (covariate.names))
+			error.message <- paste ("Bayes factor is undefined -- the dependent variable(s) ", covariate.names, " contain(s) infinity", sep = "")
```

Comment:
No

On 18 Jun 2015, at 21:33, Jonathon Love <notifications@github.com<mailto:notifications@github.com>> wrote:


In JASP-Engine/JASP/R/commonbayesianlinearmodels.R<https://github.com/jasp-stats/jasp-desktop/pull/725#discussion_r32769172>:

> @@ -86,6 +86,16 @@
>               error.message <- "Bayes factor is undefined -- the dependent variable contains infinity"
>
>       if (!exists ("error.message") && perform == "run") {
> +             covariate.names <- NULL
> +             for (covariate in options$covariates) {
> +                     if (any (!is.finite (dataset [[.v (covariate)]])))
> +                             covariate.names <- c (covariate.names, covariate)
> +             }
> +             if ( !is.null (covariate.names))
> +                     error.message <- paste ("Bayes factor is undefined -- the dependent variable(s) ", covariate.names, " contain(s) infinity", sep = "")


could this be generalised: "the variable(s) ... contains"

as i expect an infinity in the dependent or independents would be problematic.

—
Reply to this email directly or view it on GitHub<https://github.com/jasp-stats/jasp-desktop/pull/725/files#r32769172>.